plugins/source/plausible/resources/services/stats_timeseries.go

```diff
@@ -19,45 +20,41 @@ func StatsTimeseries() *schema.Table {
 			{
 				Name:        "site_id",
 				Description: "The site ID",
-				Type:        schema.TypeString,
-				CreationOptions: schema.ColumnCreationOptions{
-					PrimaryKey: true,
-				},
-				Resolver: client.ResolveSiteID,
+				Type:        arrow.BinaryTypes.String,
+				PrimaryKey:  true,
+				Resolver:    client.ResolveSiteID,
 			},
 			{
 				Name:        "date",
 				Description: "Date of the data point",
-				Type:        schema.TypeTimestamp,
-				CreationOptions: schema.ColumnCreationOptions{
-					PrimaryKey: true,
-				},
-				Resolver: ResolveDate,
+				Type:        arrow.FixedWidthTypes.Timestamp_s,
```

Comment:
This is different than our usual `Timestamp_us` approach. Initially wanted to use `Date64` but our scalar pkg doesn't support it yet.

Reply:
Why can't we use `Timestamp_us`?

Reply:
we can, but it's a date in the end. Would've used Date64 if scalar was up to date

Reply:
Even though it's a date, we don't yet have a compatibility mode, so we should try and stick to the Arrow types that we determined to be the equivalents of the old CQTypes for now (in this case `Timestamp_us`). This should guarantee that the schema doesn't break for the majority of destinations, if not all.

Reply:
(More context: this is what the struct transformer is doing now https://github.com/cloudquery/plugin-sdk/blob/main/transformers/struct.go#L307C6-L350 so if we require manual column updates we should also map it in the same way)

Reply:
ok fixed :)